src/components/Modal/index.tsx

```diff
@@ -116,7 +121,7 @@ export default function Modal({
         y: state.down ? state.movement[1] : 0,
       })
       if (state.movement[1] > 300 || (state.velocity > 3 && state.direction[1] > 0)) {
-        onDismiss()
+        onSwipe?.()
```

Comment:
Right now, I left the ability to control both dismissing (which in this context means pressing an ESC key or tapping an overlay outside of the modal) and swiping (which means dismissing a modal with a gesture). 

While this works, I wonder whether we should automatically disable dismissing when `isBottomSheet` is configured to false. In such scenario, it behaves more like a standard modal and these interactions are typical for an actual bottom sheet.

Thoughts?